### PR TITLE
`epubReadingSystem` object test anchors

### PR DESCRIPTION
**Pull request by iherman**
This is the counterpart of https://github.com/w3c/epub-tests/pull/100; to be merged only what that one is final

### CODE REVIEW

File: epub33/rs/index.html [hunk 1]
```diff
@@ -801,7 +801,7 @@ <h3>Cascading Style Sheets (CSS)</h3>
 			<section id="sec-scripted-content">
 				<h3>Scripting</h3>
 
-				<p id="confreq-rs-scripted" class="support">Reading Systems SHOULD support <a
+				<p id="confreq-rs-scripted" class="support" data-test="#scripting-support,#scripting-support_iframe,#scripting-support_svg">Reading Systems SHOULD support <a
```
Comment (dlazin) [inline epub33/rs/index.html]:
```suggestion
				<p id="confreq-rs-scripted" class="support" data-tests="#scripting-support,#scripting-support_iframe,#scripting-support_svg">Reading Systems SHOULD support <a
```

File: epub33/rs/index.html [hunk 2]
```diff
@@ -2068,7 +2068,7 @@ <h3>Description</h3>
 					<pre>alert("Reading System name: " + navigator.epubReadingSystem.name);</pre>
 				</aside>
 
-				<p>Reading Systems MUST expose the <code>epubReadingSystem</code> object on the <code>navigator</code>
+				<p id="scripting-req-epubReadingSystem" data-tests="#scripting-epubReadingSystem-support,#scripting-epubReadingSystem-support_svg,#scripting-epubReadingSystem-support_iframe,#scripting-epubReadingSystem-support_iframe_svg">Reading Systems MUST expose the <code>epubReadingSystem</code> object on the <code>navigator</code>
```
Comment (dlazin) [inline epub33/rs/index.html]: I haven't seen any other IDs that use mixed case, so I don't think we should start here. Note that line 865 is confreq-rs-scripted-readingsystem-object.

File: epub33/rs/index.html [hunk 3]
```diff
@@ -862,7 +862,7 @@ <h3>Scripting</h3>
 							provided to scripts during execution (e.g., limiting networking).</p>
 					</li>
 					<li>
-						<p id="confreq-rs-scripted-readingsystem-object">It MUST extend the <a
+						<p id="confreq-rs-scripted-readingsystem-object" data-tests="#scripting-epubReadingSystem-features,#scripting-epubReadingSystem-support,#scripting-epubReadingSystem-support_svg">It MUST extend the <a
```
Comment (dlazin) [inline epub33/rs/index.html]: I'll comment on the names more in the other PR, but a) I think we should stick with 3-letter abbreviations for consistency (I guess scr-) and b) as below, I don't think we should introduce mixed case in names.

I recommend #scr-readingsystem-features, etc.

Applies throughout.

**Comment by iherman** [comment]
@dlazin I have made the test names and have also updated the anchors in here.

**Review by dlazin — APPROVED** [review]
Thank you! I'm sorry to be so annoying about the names :)

**Pull request merged**